packages/terra-consumer-nav/src/components/user-profile/ProfileLink.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import classNames from 'classnames/bind';
+import navElementShape from '../../NavPropShapes';
+import SmartLink from '../smart-link/SmartLink';
+import styles from './UserProfile.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = { ...navElementShape };
+
+const ProfileLink = ({
+  text,
+  url,
+  icon,
```

Comment:
should icon/badgeValue be removed?

Reply:
will remove icon, badgeValue and isActive